server/layout/html.js

```diff
@@ -41,7 +41,7 @@ const render = ({
 </head>
 
 <body class="govuk-template__body ">
-
+    <script>document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
```

Comment:
You could make this shorter if you wanted:
```suggestion
    <script>document.body.className = (document.body.className ? document.body.className : '') + ' js-enabled';</script>
```